src/renderer/features/workbench/pages/report/components/LicensesObligations.tsx

```diff
@@ -22,6 +22,17 @@ const useStyles = makeStyles({
 export default function MatchesForLicense({ data }) {
   const classes = useStyles();
 
+  const init = () => {
+    const aux = data.reduce((acc, curr) => {
+      if (!acc[curr.label]) acc[curr.label] = curr;
+      return acc;
+    }, {});
+
+    return aux;
+  };
+
+  const licenseHash = init();
```

Comment:
@agustingroh best solutions is trigger this init on mounted (useEffect(..., []) and set "licenseHash" into component state. Otherwise each render init function is triggered.